cli/crates/cli/tests/reserved_dates.rs

```diff
@@ -0,0 +1,100 @@
+mod utils;
+
+use crate::utils::consts::{
+    RESERVED_DATES_CREATE_TODO, RESERVED_DATES_CREATE_TODO_LIST, RESERVED_DATES_NESTED_CREATION, RESERVED_DATES_SCHEMA,
+};
+use chrono::{DateTime, Duration, Utc};
+use serde_json::{json, Value};
+use utils::environment::Environment;
+
+#[test]
+fn reserved_dates() {
+    // TODO: Create simpler client setup (one-line)
+    let mut env = Environment::init(4016);
```

Comment:
The init templates PR adds a test on `4016`, until we have a better solution for this, would it be possible to start with `4017`?

Reply:
Yeah, I'll fix those once the gateway PRs have been merged when rebasing. If you haven't merged your PR by then, I'll start from 4017.